src/core/Comptroller.sol

```diff
@@ -1244,6 +1246,14 @@ contract Comptroller is
         return uint256(Error.NO_ERROR);
     }
 
+    /**
+     * @notice Sets the global pause borrows override
+     */
+    function _setGlobalPauseBorrowOverride(bool status) external {
+        require(hasAdminRights(), "!admin");
```

Comment:
this isn't fully trustless because the Fuse admin could just set this to false again right?

Reply:
it gives them the option to just toggle it when global borrows are paused, though I could make this just callable for comptroller admin, I'm not sure that's necessary